Dockerfile

```diff
@@ -1,11 +1,103 @@
-FROM nfnty/arch-npm
+FROM opensuse:tumbleweed
 MAINTAINER Lasse Schuirmann lasse.schuirmann@gmail.com
 
-RUN pacman -Syu ruby php clang indent mono python python-pip python-setuptools git go wget --noconfirm
+RUN zypper addrepo -f \
+  http://download.opensuse.org/repositories/devel:/languages:/lua/openSUSE_Factory/ \
+  devel:languages:lua
 
-RUN wget https://raw.githubusercontent.com/coala-analyzer/coala-bears/master/package.json
+RUN zypper --no-gpg-checks update --no-confirm && zypper install --no-confirm \
+  cppcheck \
+  curl \
+  espeak \
+  dbus-1-python3 \
+  git \
+  go \
+  hlint \
+  indent \
+  java \
+  julia \
+  libclang \
+  lua \
+  lua-devel \
+  luarocks \
+  m4 \
+  mono \
+  npm \
+  perl \
+  perl-Perl-Critic \
+  php \
+  python3 \
+  python3-gobject \
+  python3-pip \
+  python3-setuptools \
+  R-base \
+  ruby \
+  tar \
+  texlive-chktex \
+  unzip \
+  verilator \
+  wget
+
+# Coala setup and python deps
+RUN pip install --upgrade pip
+RUN mkdir /tmp/coala
+WORKDIR /tmp/coala
+ADD coala-requirements.txt coala-deps.txt
```

Comment:
Well, we should at least not pull them from the master branch as that one 
can dynamically change while the Docker images contains the latest Release 
from PyPi.

Pulling them from a Release branch would be better but it's easy to forget 
to update them.

Anyway I have not been consistent with that yet as I probably pull stuff 
from Master in other places.